src/main/java/cart/controller/ProductController.java

```diff
@@ -0,0 +1,50 @@
+package cart.controller;
+
+import cart.domain.Product;
+import cart.dto.request.ProductSaveRequest;
+import cart.dto.request.ProductUpdateRequest;
+import cart.service.ProductService;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProductController {
+
+    private final ProductService productService;
+
+    public ProductController(ProductService productService) {
+        this.productService = productService;
+    }
+
+    @GetMapping("/product")
```

Comment:
```suggestion
    @GetMapping("/products")
```
url은 리소스를 나타내기도 해서요. 보통 복수형 명사를 사용하는 것 같습니다! get API도 products의 컬렉션을 조회하고 있으니까요!

Reply:
url path부분에 복수형이 오는 것이 더 대중적일 거라는 것에 동의합니다!
api 설계에 대해서는 더 공부해보아야 할 것 같습니다. 제안 주신대로 수정하겠습니다